go/libraries/doltcore/table/editor/table_edit_accumulator_test.go

```diff
@@ -164,30 +164,30 @@ func TestGet(t *testing.T) {
 	key5 := newTuple(t, types.Int(5))
 	key6 := newTuple(t, types.Int(6))
 
-	// test uncommitted
+	// test uncomitted
```

Comment:
This change is going from the correct spelling to an incorrect spelling. Please revert this file.